src/models/elan1-r1/README.md

```diff
@@ -0,0 +1,29 @@
+# Eland 1U (elan1-r1)
+
+![Eland 1U](./img/elan1-r1-frontports.avif)
```

Comment:
GitHub previews, GNOME Image Viewer, and Gwenview all don't recognize .avif files, which is kind of annoying. If you remove `?auto=compress,format` from the end of the Prismic image URL from the sames page, then Firefox should save it as a .png, which you can then save in .webp for compression. (Also needs to cropped/resized down to 1920 width.)

Reply:
Good point, that should be updated. I guess I can leave the lfs inclusion .gitignore, just in case a .avif gets uploaded in the future.